ddtrace/_monkey.py

```diff
@@ -118,6 +118,8 @@
     "futures": ("concurrent.futures",),
     "vertica": ("vertica_python",),
     "aws_lambda": ("datadog_lambda",),
+    # patch httplib for PY2, http.client for PY3
+    "httplib": ("httplib", "http.client"),
```

Comment:
```suggestion
    "httplib": ("httplib" if PY2 else "http.client",),
```
to avoid registering a hook on a module path that will never occur?